packages/adapter-cloudflare/package.json

```diff
@@ -28,6 +28,7 @@
 	"types": "index.d.ts",
 	"files": [
 		"files",
+		"templates-experimental",
```

Comment:
why make a new directory for this new file instead of just putting it in `files`?

Reply:
The 'files' was mentioned in the `.gitignore` and used as the build process output folder.  I have moved the `.assetsignore` to the `files` folder and updated `.gitignore` accordingly.